test/cases/parsing/harmony-tdz/index.js

```diff
@@ -1,8 +1,16 @@
 import value, { exception } from "./module";
 
-it("should have a TDZ for exported const values", function() {
-	expect((typeof exception)).toBe("object");
+it("should have a TDZ for exported const values", () => {
+	expect(typeof exception).toBe("object");
 	expect(exception).toBeInstanceOf(Error);
-	expect(exception.message).toMatch(/ is not defined$/);
+
+	if (["12"].includes(process.versions.node.split(".")[0])) {
```

Comment:
You can't access `process.versions` in the bundled code.

I think it's fine to create a regexp to match both.

``` js
expect(exception.message).toMatch(/ is not defined$|^Cannot access '.+?' before initialization$/);
```